### DISCUSSION TITLE
Searxng and gunicorn?

### DISCUSSION THREAD

**Discussion by ghost** [searxng/searxng#1528] [Ideas]
Just wondering, should it be possible to use gunicorn instead of uwsgi? And if yes, could you please give me a detailed explanation of how to do this? Gunicorn seems to be more system agnostic and it would be theorically easier to get searxng working in distros that don't run systemd. I found [this guide ](https://www.mavjs.org/post/setting-up-searx-with-gunicorn-and/) though it's a bit old and I don't really know if it might work. Thanks in advance.

**Comment by return42** [comment]
> in distros that don't run systemd

For my information, what distros do not run systemd?

**Reply by ghost** [reply]
For example Void, Artix, Devusn, Obarun/Joborun, PCLinuxOS, Alpine, to quote the most famous ones. There are different alternative init softwares like runit, 66,SysV, openRC.
